src/util/dom.js

```diff
@@ -58,17 +58,15 @@ export function contains(parent, child) {
   } while (child = child.parentNode)
 }
 
-export let activeElt = function() {
-  let activeElement = document.activeElement
-  while (activeElement && activeElement.root && activeElement.root.activeElement)
-    activeElement = activeElement.root.activeElement
-  return activeElement
-}
-// Older versions of IE throws unspecified error when touching
-// document.activeElement in some cases (during loading, in iframe)
-if (ie && ie_version < 11) activeElt = function() {
-  try { return document.activeElement }
-  catch(e) { return document.body }
+export function activeElt() {
+  // Older versions of IE and sometimes Edge throws unspecified error when touching
+  // document.activeElement in some cases (during loading, in iframe)
+  try {
+    let activeElement = document.activeElement
```

Comment:
Does it still work when you only wrap the first access to `document.activeElement` in a `try`? Non-trivial code wrapped in error-swallowing try/catch scares me, since when someone introduces a bug there, we probably won't notice.